dagshub/upload/wrapper.py

```diff
@@ -271,17 +272,10 @@ def _log_upload_details(self, data, res, files):
             f"Data:\n{json.dumps(data, indent=4)}\n"
             f"Files:\n{json.dumps(list(map(str, files)), indent=4)}"
         )
-        try:
-            content = json.dumps(res.json(), indent=4)
-        except Exception:
-            content = res.content.decode("utf-8")
 
         if res.status_code != HTTPStatus.OK:
-            raise RuntimeError(f"Response ({res.status_code}):\n" f"{content}")
+            raise determine_error(res)
```

Comment:
If we use `determine_error` this way I think it should be renamed to something along the lines of `upload_api_exception`.